envoy/server_test.go

```diff
@@ -282,12 +294,14 @@ func Test_PortForServicePort(t *testing.T) {
 					state.AddServiceEntry(anotherHTTPSvc)
 					<-snapshotCache.Waiter
 
-					resources := envoyMock.GetResource(stream, cache.ClusterType, state.Hostname)
+					resources := envoyMock.GetResource(stream, cache.EndpointType, state.Hostname)
 					So(resources, ShouldHaveLength, 1)
-					endpoints := extractClusterEndpoints(resources[0], httpSvc)
-					So(endpoints, ShouldHaveLength, 2)
+					loadAssignment := &api.ClusterLoadAssignment{}
```

Comment:
I see that var name throughout. Isn't is just `assignment`? I'm good with it like this to make progress, just seems like a weird name.